static/app/views/organizationGroupDetails/allEventsTable.tsx

```diff
@@ -38,8 +38,6 @@ const AllEventsTable = (props: Props) => {
     'user.display',
     ...(isPerfIssue ? ['transaction.duration'] : []),
     'timestamp',
-    'attachments',
-    'minidump',
```

Comment:
@DominikB2014 do you know if there's anything else relying in fields?

Reply:
Perhaps the table itself relies on these fields to display the columns

Reply:
The code can definetly be much more generic to not behave like that

Reply:
Yeah, I tried to reused the allEvents table from performance, its a side affect it not being generic enough to support joining attachments and discover data (definetly will want to improve it but its a bit of a refactor) I just tested removing the attachments from the query, and the columns disappear :(

Reply:
sigh yeah but we can't be using attachments and minidumps as field in discover either 
kk I can work on splitting that logic apart

Reply:
Just looked into the code, the existence of a column comes from columnOrder which comes from eventView
https://github.com/getsentry/sentry/blob/bb6d1544ffb486a0712a3a327a37960916b4501f/static/app/views/performance/transactionSummary/transactionEvents/eventsTable.tsx#L289,
so a temp fix would be to remove the fields in the eventView and update the columnOrder for custom columns like attachments

The actual title column comes is set here, and is defaulted to the column name in the column order object, if the column name isn't set with `columnTitles`
https://github.com/getsentry/sentry/blob/0a2c9c5e03245cf8b99dac66c97de02c261bf56d/static/app/views/performance/transactionSummary/transactionEvents/eventsTable.tsx#L292